data/jobs.txt

```diff
@@ -1262,6 +1262,302 @@ mission "Escort (Extra Large, Southern, Dangerous Origin or Destination)"
 		dialog "The captain of the <npc> thanks you for escorting them safely, and pays you <payment>."
 	on visit
 		dialog "You have reached <planet>, but you left the <npc> behind! Better depart and wait for them to arrive in this star system."
+		
+mission "Rescue (Interceptor, Northern)"
+	name "Rescue Merchant Interceptor"
+	description "The captain of the <npc> bit off more than they could chew. Repair them and escort them safely back to <destination>."
```

Comment:
Here it says `the captain of the <npc>` but below it says `The captain of <npc>`. I suggest going with whichever sounds better with ship names. This applies to all the merchant missions here. The military ones use the second version.